### ISSUE TITLE
MShockXotto+ is missing its fallback.png file

### ISSUE THREAD

**Issue by tinwhiskers2** [I-am-Erk/CDDA-Tilesets#1995]
Error: could not load image /home/xxx/.local/share/cataclysm-dda/gfx//MShockXotto+/fallback.png

The file is indeed missing from that location on github.

**Comment by Fris0uman** [comment]
True, but MSX already comes with the game and falback.png should be there. We can consider adding fallback to our release to make them stand alone anyway, but right now the fallback files should be with the game
